server/api/appointment_handler.py

```diff
@@ -77,5 +78,31 @@ def get_appointment(uid):
         .filter(EventTypes.user_id == uid).all()
     return jsonify([appointments.to_dict() for appointments in all_appointments])
 
+@appointment_handler.route('/appointment/<int:id>', methods=['DELETE'])
+@check_token
+def delete_appointment(id):
+    try:
+        user = Users.query.get(int(request.get_json().get("user_id")))
+        appointment_to_delete = Appointments.query.get(id)
```

Comment:
can you change the id in `/appointment/<int:id>` as User ID, and get the `appointment IDs` in request body?
Because I'm building a table to display the appointments, and there is a possibility for users to delete more than one appointment at one time. And we may build a loop here to delete a list of appointment.

![image](https://user-images.githubusercontent.com/70559690/113108561-f4fb0180-91b9-11eb-8991-8fe2fcaa7526.png)

I think this gives a better UX

Reply:
If you don't have time to change the code, I can change it later after the merge 😃

Reply:
ok sure! Are we passing a list of appointment ids to the backend and loop there, or are we making multiple calls on the frontend?

I guess this is just a guideline and we can change depending on different situations, but according to RESTful routing convention, normally the route is just /appointment/Appointment_id (https://medium.com/@atingenkay/restful-routes-what-are-they-8fe221521bb)

Reply:
I made them as passing a list of IDs ex.` ['11111111',  '2222222', '3333333']` 
just do what is convenient for you and let me know later